frontend/src/utils/aboutData.ts

```diff
@@ -4,6 +4,87 @@ export const aboutText = [
   'The code is licensed under the MIT License, encouraging contributions while protecting the authors from legal claims. All OWASP Nest leaders are certified ISC2 professionals and OWASP members who adhere to the OWASP Code of Conduct.',
 ]
 
+export const newAboutContent = {
```

Comment:
I think we should split it into smaller bits. Maybe combine `Mission` and `Who It's For`, then have `Key Features`, `Gt Involved` and `Project History` as separate items.